aide_design/designs/civil_work/floc.py

```diff
@@ -0,0 +1,236 @@
sembly:\n  BottomBaffle: {Height: !DP '1.728 meter', Thickness: !DP '2 millimeter', Width: !DP '0.3134\n      meter'}\n  EntTank_Length: !DP '2.2 meter'\n  Length: !DP '7.35 meter'\n  Num_Exit: !DP '18 '\n  Num_Inlet: !DP '26 '\n  Spacing: !DP '0.272 meter'\n  Thickness: !DP '2 millimeter'\n  TotalNum: !DP '2 '\n  WallThickness: !DP '0.15 meter'\n  Width: !DP '0.3134 meter'\nConcreteChannels:\n  Channel: {EntTank_Length: !DP '2.2 meter', FirstLength: !DP '5.786 meter', FloorThickness: !DP '0.2\n      meter', Height: !DP '2.5 meter', Length: !DP '7.35 meter', TotalNum: !DP '2 ',\n    WallThickness: !DP '0.15 meter', Width: !DP '0.3134 meter'}\n  EntTank_Length: !DP '2.2 meter'\n  EvenWall: {EntTank_Length: !DP '2.2 meter', FirstLength: !DP '5.786 meter', FloorThickness: !DP '0.2\n      meter', Height: !DP '2.5 meter', Length: !DP '7.35 meter', TotalNum: !DP '2 ',\n    WallThickness: !DP '0.15 meter', Width: !DP '0.3134 meter'}\n  FirstChannel: {EntTank_Length: !DP '2.2 meter', FirstLength: !DP '5.786 meter',\n    FloorThickness: !DP '0.2 meter', Height: !DP '2.5 meter', Length: !DP '7.35 meter',\n    TotalNum: !DP '2 ', WallThickness: !DP '0.15 meter', Width: !DP '0.3134 meter'}\n  FirstLength: !DP '5.786 meter'\n  FloorThickness: !DP '0.2 meter'\n  Height: !DP '2.5 meter'\n  LastChannel: {EntTank_Length: !DP '2.2 meter', FirstLength: !DP '5.786 meter', FloorThickness: !DP '0.2\n      meter', Height: !DP '2.5 meter', Length: !DP '7.35 meter', TotalNum: !DP '2 ',\n    WallThickness: !DP '0.15 meter', Width: !DP '0.3134 meter'}\n  Length: !DP '7.35 meter'\n  OddWall: {EntTank_Length: !DP '2.2 meter', FirstLength: !DP '5.786 meter', FloorThickness: !DP '0.2\n      meter', Height: !DP '2.5 meter', Length: !DP '7.35 meter', TotalNum: !DP '2 ',\n    WallThickness: !DP '0.15 meter', Width: !DP '0.3134 meter'}\n  TotalNum: !DP '2 '\n  WallThickness: !DP '0.15 meter'\n  Width: !DP '0.3134 meter'\nEntFlocBaffleSupport:\n  BottomBaffle: {Height: !DP '1.728 meter', Thickness: !DP '2 millimeter', Width: !DP '0.3134\n      meter'}\n  TopBaffle: {Height: !DP '2.228 meter', Thickness: !DP '2 millimeter', Width: !DP '0.3134\n      meter'}\n  bafflethickness: !DP '2 millimeter'\n  numberbaffles: !DP '18 '\n  numberentbaffles: !DP '26 '\nMainFlocBaffleSupport: {bafflethickness: !DP '2 millimeter', numberbaffles: !DP '18 ',\n  numberentbaffles: !DP '26 '}\nObstacles_Assembly:\n  Num_Exit: !DP '26 '\n  Num_Inlet: !DP '18 '\n  Obstacle: {Width: !DP '0.3134 meter'}\n  Spacing: !DP '0.272 meter'\n  Thickness: !DP '2 millimeter'\n  TotalNum: !DP '2 '\n  WallThickness: !DP '0.15 meter'\n  Width: !DP '0.3134 meter'\nTopBaffles_Assembly:\n  EntTank_Length: !DP '2.2 meter'\n  Length: !DP '7.35 meter'\n  Spacing: !DP '0.272 meter'\n  Thickness: !DP '2 millimeter'\n  TopBaffle: {Height: !DP '2.228 meter', Thickness: !DP '2 millimeter', Width: !DP '0.3134\n      meter'}\n  TotalNum: !DP '2 '\n  WallThickness: !DP '0.15 meter'\n  Width: !DP '0.3134 meter'\nnumberrows: !DP '1 '\n"
```

Comment:
How will we format this? The evaluation line is quite long and unruly to handle. Perhaps we can honor this using the new line characters?

Reply:
We discussed this on Friday. I'm not sure what we decided the solution was though.